daisy_workflows/image_build/debian/debian_sid_fai.wf.json

```diff
@@ -3,6 +3,7 @@
   "Vars": {
     "build_date": {"Value": "${DATE}", "Description": "Build datestamp used to version the image."},
     "image_dest": {"Required": true, "Description": "The GCS path for the destination image tar.gz."},
+    "destination": {"Required": true, "Description": "The GCS path for the destination image metadata json"},
```

Comment:
Looks like there are some more `destination` daisy variables, eg: daisy_workflows/image_build/enterprise_linux/rhel_6.wf.json

Reply:
oh. I thought you just want to change in debian. For EL, there is no "image_dest" var, instead it's named "destination" and it could be reused in different step. So It's not necessary to rename it as it's just used in export_metadata step. The meaning is clear in that step. Yea, that's another diverge from debian.